content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -294,11 +296,12 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
-#### Role Examples
+#### Role examples
 
-Only the `rules` section is shown in the following examples.
+The following examples are Role excerpts that only
```

Comment:
this isn't quite accurate, some of the examples are only applicable to ClusterRole objects